specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/examples/Patches_Apply.json

```diff
@@ -0,0 +1,46 @@
+{
+    "operationId": "Patches_Apply",
+    "parameters": {
+      "subcriptionId": "00000000-0000-0000-0000-000000000000",
+      "resourceGroupName": "rg",
+      "builderName": "testBuilder",
+      "patchName": "testPatch-25fe4b",
+      "api-version": "2023-05-02-preview",
+      "patchEnvelope": {
+        "configuration": {
+          "patchId": "1ce4b2a0"
+        }
+      }
+    },
+    "responses": {
+      "200": {
+        "body": {
+          "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/builders/testBuilder/patches/testPatch-25fe4b",
+          "name": "testPatch",
```

Comment:
nit: there are a couple of places in this example that use `testPatch` rather than `testPatch-25fe4b` from the original call